craft_parts/plugins/rust_plugin.py

```diff
@@ -254,9 +246,12 @@ def get_build_commands(self) -> List[str]:
                     pushd "{crate}"
                     cargo build --workspace --release {config_cmd_string}
                     # install the final binaries
-                    # TODO(liushuyu): this will also install proc macros in the workspace,
-                    # which the user may not want to keep in the final installation
                     find ./target/release -maxdepth 1 -executable -exec install -Dvm755 {{}} "{self._part_info.part_install_dir}" ';'
+                    # remove proc_macro objects
```

Comment:
Is there any reason someone might want to keep those? (e.g. during debugging the snap)

I can't come up with one, but if there is we might want to make a way to prevent this from running.

Reply:
> Is there any reason someone might want to keep those? (e.g. during debugging the snap)
> 
> I can't come up with one, but if there is we might want to make a way to prevent this from running.

No. Debugging a Rust application Snap also does not need the compiler plugins.
Compiler plugins are only used to generate code during the compilation time.